### PR TITLE
Remove all links to generic vision ancestry features

### PR DESCRIPTION
**Pull request by TikaelSol**
Wish we had just deleted this thing back in 0.8 when we stopped using it.

### CODE REVIEW

File: packs/spell-effects/spell-effect-evolution-surge-sight.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Evolution Surge (Sight)",
     "system": {
         "description": {
-            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</em></p>\n<p>You flood your eidolon with power, creating a temporary evolution in your eidolon's capabilities. Choose one of the following effects:</p>\n<ul>\n<li>Your eidolon gains @UUID[Compendium.pf2e.ancestryfeatures.Item.Low-Light Vision] and @UUID[Compendium.pf2e.ancestryfeatures.Item.Darkvision].</li>\n</ul>"
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</em></p>\n<p>You flood your eidolon with power, creating a temporary evolution in your eidolon's capabilities. Choose one of the following effects:</p>\n<ul>\n<li>Your eidolon gains @UUID[Compendium.pf2e.ancestryfeatures.Item.Low-Light Vision] and darkvision.</li>\n</ul>"
```
Comment (simonward) [inline packs/spell-effects/spell-effect-evolution-surge-sight.json:7]: Perhaps @UUID[Compendium.pf2e.ancestryfeatures.Item.Low-Light Vision] should be stripped as well?

Comment (TikaelSol) [reply packs/spell-effects/spell-effect-evolution-surge-sight.json:7]: and greater darkvision

File: packs/backgrounds/feral-child.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
...
+            "value": "<p>You spent your youth in the wilderness, living close to or perhaps raised by animals. You have a close, mystical connection with these animals and gained certain abilities from them, though this limited your well-roundedness in mental pursuits.</p>\n<p>Choose one ability boost. It must be <strong>Strength</strong>, <strong>Dexterity</strong>, or <strong>Constitution</strong>.</p>\n<p>You are trained in Nature and Survival. You gain low-light vision (or darkvision if you already had Low-Light Vision), imprecise scent with a range of 30 feet, and the @UUID[Compendium.pf2e.feats-srd.Item.Forager] skill feat.</p>"
```
Comment (simonward) [inline packs/backgrounds/feral-child.json:19]:
```suggestion
            "value": "<p>You spent your youth in the wilderness, living close to or perhaps raised by animals. You have a close, mystical connection with these animals and gained certain abilities from them, though this limited your well-roundedness in mental pursuits.</p>\n<p>Choose one ability boost. It must be <strong>Strength</strong>, <strong>Dexterity</strong>, or <strong>Constitution</strong>.</p>\n<p>You are trained in Nature and Survival. You gain low-light vision (or darkvision if you already had low-light vision), imprecise scent with a range of 30 feet, and the @UUID[Compendium.pf2e.feats-srd.Item.Forager] skill feat.</p>"
```

File: packs/feats/elemental-apotheosis.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You take on an elemental countenance, appearing like a type of elemental of your element. You gain any Speeds listed in the <em>@UUID[Compendium.pf2e.spells-srd.Item.Elemental Form]</em> spell for your element and gain <em>@UUID[Compendium.pf2e.ancestryfeatures.Item.Darkvision]</em> if you don't already have it. You also gain the elemental trait and the trait of your chosen element.</p>"
+            "value": "<p>You take on an elemental countenance, appearing like a type of elemental of your element. You gain any Speeds listed in the <em>@UUID[Compendium.pf2e.spells-srd.Item.Elemental Form]</em> spell for your element and gain <em>darkvision</em> if you don't already have it. You also gain the elemental trait and the trait of your chosen element.</p>"
```
Comment (simonward) [inline packs/feats/elemental-apotheosis.json:14]:
```suggestion
            "value": "<p>You take on an elemental countenance, appearing like a type of elemental of your element. You gain any Speeds listed in the @UUID[Compendium.pf2e.spells-srd.Item.Elemental Form] spell for your element and gain darkvision if you don't already have it. You also gain the elemental trait and the trait of your chosen element.</p>"
```

**Pull request merged**